src/components/delegations/DelegationList/index.tsx

```diff
@@ -13,9 +9,10 @@ import { Amount } from "./components/Amount";
 import { Status } from "./components/Status";
 import { TxHash } from "./components/TxHash";
 
-const columns = (
-  handleActionClick: (action: string, txHash: string) => void,
-): TableColumn<DelegationV2, DelegationV2Params>[] => [
+const columns = (): TableColumn<
```

Comment:
I don't think you need this function anymore @jrwbabylonlab. You can keep columns as regular array